cmd/kgctl/connect_other.nogo

```diff
@@ -0,0 +1,20 @@
+// +build !linux
```

Comment:
This also, no?

Reply:
yes, i was looking for that file. I thought i deleted it!!

Reply:
Agh that would have been a bummer 🙀